tests/data/import.csv

```diff
@@ -0,0 +1,3 @@
+IBAN;Auszugsnummer;Buchungsdatum;Valutadatum;Umsatzzeit;Zahlungsreferenz;Waehrung;Betrag;Buchungstext;Umsatztext
+ATXYZ123400000;1;2017-02-14;2017-02-14;2017-02-14-11.57.44.235044;"";EUR;-100,00;"Bankomat";" BANKOMAT  00000483 K2 UM 11:56"

```

Comment:
What exactly do you mean? Lines in the CSV-file that would correspond to/generate `note`s in a beancount ledger?